ReflectometryServer/beam_path_calc.py

```diff
@@ -2,6 +2,9 @@
 Objects to help with calculating the beam path when interacting with a component. This is used for instance for the
 set points or readbacks etc.
 """
+from __future__ import division
```

Comment:
This is not required because Python 3 will do true division anyways. As for line `626` and `529` in this module we can make the divisor an `int`.